Sources/FluentUI_iOS/Components/Command Bar/CommandBarItem.swift

```diff
@@ -5,7 +5,22 @@
 
 import UIKit
 
-public typealias CommandBarItemGroup = [CommandBarItem]
+@objc(MSFCommandBarItemGroup)
+public class CommandBarItemGroup: NSObject {
+    public let items: [CommandBarItem]
+    public let label: String?
+
+    @objc public init(items: [CommandBarItem]) {
+        self.items = items
+        self.label = nil
+    }
+
+    @available(*, deprecated, message: "CommandBarItemGroup labels are experimental.")
```

Comment:
If this is deprecated, downstream consumers won't be able to use this API at all without disabling *all* deprecation errors. We can probably make do with a simple warning in its documentation comment (which, by the way, we should probably do anyway).